### ISSUE TITLE
LazyVerticalGrid scroll: "IllegalStateException: Unknown Modifier.Node type"

### ISSUE THREAD

**Issue by taaeng** [JetBrains/compose-multiplatform#3559]
**Describe the bug**
Crash when scrolling in a LazyVerticalGrid using OutlinedTextField with alternating enabled flag set.

Happens in 1.5.0-rc02, not 1.4.3.

**Affected platforms**
Desktop

**Versions**
- Kotlin version*: 1.9.0
- Compose Multiplatform version*: 1.5.0-rc02
- OS version(s): Mac OS 13.5 (22G74) 
- OS architecture (x86 or arm64):  M1 arm64
- JDK (for desktop issues): 11

**To Reproduce**
1. Scroll in LazyVerticalGrid

```
@Composable
@Preview
fun App() {
    MaterialTheme {
        LazyVerticalGrid(
            columns = GridCells.Fixed(4),
            modifier = Modifier.fillMaxSize()
        ) {
            items((0 until 200).toList()) { item ->
                OutlinedTextField(
                    value = "Item: $item",
                    enabled = item % 2 == 0,
                    onValueChange = {},
                )
            }
        }
    }
}
```

**Expected behavior**
No crash

**Screenshots**
<img width="798" alt="Screenshot 2023-08-24 at 13 11 52" src="https://github.com/JetBrains/compose-multiplatform/assets/1119881/3f5b4438-a8dc-4e20-a1e5-42d62681e261">

**Comment by igordmn** [comment]
[Jetpack Compose issue](https://issuetracker.google.com/issues/297374472)

**Comment by igordmn** [comment]
Will be fixed in 1.5.0-rc05

**Comment by okushnikov** [comment]
Please check the <a href="https://youtrack.jetbrains.com/issue/CMP-3559">following ticket</a> on YouTrack for follow-ups to this issue. GitHub issues will be closed in the coming weeks.
